src/components/Contact/ContactCard2.jsx

```diff
@@ -1,59 +1,99 @@
+            sx={{
+                textAlign: "left !important",
+                padding: "1.5rem !important",
+                marginTop: "0 !important",
+                height: "auto !important",
+                borderRadius: "1rem",
+                width: "350px",
+                boxShadow: "0px 4px 12px rgba(0, 0, 0, 0.1)",
+                backgroundColor: theme.palette.mode === "dark" ? "rgba(255,255,255,0.05)" : "white",
+                border: `1px solid ${theme.palette.mode === "dark" ? "rgba(255,255,255,0.12)" : "white"}`,
+                backdropFilter: "blur(2.4px)",
+            }}
+        >
+            <ServiceItemBox
                 sx={{
-                    padding: "1rem",
-                    width: "80%",
-                    '@media (max-width: 600px)': {
-                        padding: "0.5rem",
-                    },
+                    alignItems: "flex-start !important",
+                    gap: "1rem",
                 }}
             >
-                <CardHeading
+                <Box
                     sx={{
                         display: "flex",
-                        gap: 1,
-                        fontSize: { xs: '1.5rem', sm: '1.75rem', md: '2rem' }
+                        alignItems: "center",
+                        gap: "1.5rem !important",
+                        paddingBottom: "2rem",
                     }}
-                    special
                 >
-                    <CardImage src="/howitworkicons/appointment.gif" alt="Location" width={40} height={40} style={{ padding: "5px" }} />
-                    WERKTIJDEN
-                </CardHeading>
+                    <ServiceItemIconContainer
+                        sx={{
+                            width: "5rem !important",
+                            height: "5rem !important",
+                            marginBottom: "0 !important",
+                        }}
+                    >
+                        <ServiceItemIcon
+                            src="/howitworkicons/appointment.gif"
+                            alt="Appointment Icon"
+                            width={15}
+                            height={15}
+                            sx={{ width: "60% !important", height: "60% !important" }}
+                        />
+                    </ServiceItemIconContainer>
 
-                <Typography
-                    variant="h5"
-                    component="div"
-                    sx={{ fontSize: { xs: '0.875rem', sm: '1rem', md: '1.25rem' } }}
-                >
+                    <ServiceItemHeading
+                        sx={{
+                            fontSize: "1.6rem !important",
+                            marginTop: "1.25rem",
+                            paddingBottom: "0.5rem",
+                        }}
+                    >
+                        WERKTIJDEN
+                    </ServiceItemHeading>
+                </Box>
+
+                {/* Description */}
```

Comment:
Remove unnecessary comments like these, since the component names themselves speak for what is happening